docs/dev-docs-for-loader-customization.rst

```diff
@@ -0,0 +1,69 @@
+.. _loader_customization_guide:
+
+===============================
+Data Loader Customization Guide
+===============================
+
+The `glue` data visualization tool is highly customizable for different data types, such as for FITS files
+that are often used in solar physics research. There are two ways to customize `glue` using the `glue-solar` plugin:
+
+1. The first option is to develop a custom loader for the solar data taken from practically any (ground-based or
+satellite-borne) instrument.
+
+2. The second option is to develop a custom viewer to enable a certain very specific way to view your data, such as
+an alternative of the existing 2D image viewer that presents your data in a manner you like; e.g. color-coding the
+lines or data points.
+
+While these two options serve different needs, the former is often needed to load your pre-processed data, as `glue`
+already has the functionality to display the typical FITS file (with extensions ``.fits``, ``.fit``, or ``.ft``), such
+as the existing basic 2D image viewer and 1D profile viewer (useful for viewing 1D spectra). Hence in this guide, we
+walk through how to construct such a custom data loader for your solar physics use cases.
+
+The Case of the IRIS subpackage
+-------------------------------
+
+If we look at the ``glue_solar/instruments/iris`` directory, we can find (at least at the time of writing of this
+guide) the files within follows:
+
+1. __init__.py: This is the typical ``init`` file that is required for running unit tests with ``pytest``.
```

Comment:
I am not sure this is useful information?

Reply:
It's chiefly for those who are new to testing with the `pytest` framework

Reply:
An init file isn't only for pytest tho.

Reply:
True... Let me add to it